### PR TITLE
Swap to diskless upload, introduce 3dgrids

### PR DESCRIPTION
**Pull request by daniel-sol**
* Refactor to allow for both tables and cpgrid with cpgrid_properties
* Change to diskless upload

Solving issue #48 and #47

### CODE REVIEW

**Comment by perolavsvendsen** [comment]
Suggest we do a demo/review of this, it is a huge PR. I didn't quite understand what is going on...

File: src/fmu/sumo/sim2sumo/grid3d.py [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+from .common import (
+    generate_meta,
+    convert_to_bytestring,
+    convert_2_sumo_file,
+    fix_suffix,
+)
+
+
+def xtgeo_2_bytes(obj):
+    """Convert xtgeo object to bytesring
```
Comment (equinor-ruaj) [inline src/fmu/sumo/sim2sumo/grid3d.py:29]: Should be "Convert xtgeo object to bytes" ?

**Comment by equinor-ruaj** [comment]
Tests are failing because they all use the same `case_uuid`.
Had no issues when either running for a single python version: https://github.com/equinor/fmu-sumo-sim2sumo/pull/53/commits/c55ad0713f3a225c8a46d3db801ec48be99dde93
Or if setting `max-parallel: 1`: https://github.com/equinor/fmu-sumo-sim2sumo/pull/53/commits/b20ec7644d6fd4eacd5b97e98eac74dde062af75

There was a similar issue with the uploader tests. Fixed by creating a new uuid at runtime: https://github.com/equinor/fmu-sumo-uploader/commit/b422bc04a738ef434b965369f29e395d32060e92

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+def set_up_tmp(path):
+    reek_tmp = path / "reek_tmp"
+    shutil.copytree(REEK_ROOT, reek_tmp, copy_function=shutil.copy)
+    real0 = reek_tmp / "realization-0/iter-0"
+    config_path = real0 / "fmuconfig/output/global_variables.yml"
+    os.chdir(real0)
+    eight_datafile = real0 / "eclipse/model/EIGHTCELLS.DATA"
+    return real0, eight_datafile, config_path
```
Comment (equinor-ruaj) [inline tests/conftest.py:37]: Why do we have to put things in a `tmp` folder?

**Review by equinor-ruaj — APPROVED** [review]
👍

**Pull request merged**